go/config/README.md

```diff
@@ -0,0 +1,48 @@
+All go processes in the TEN ecosystem will use the config mechanism here to load their configuration.
+
+Ten config has a hierarchical structure, so every field has a unique field name. Some fields are used by multiple processes, 
+while others are only used by a single process.
+
+For example, `network.chainId` is a field that might be used by multiple processes, but `host.rpc.httpPort` is only used by the host processes.
+
+The fields are defined in a single place, in the Config structs declared in `config.go`. These declarations include their type,
+the mapstructure annotation that gives their yaml field name and any comments about their usage.
+
+This library provides a loading mechanism powered by viper, it reads 0-base-config.yaml to initialise the config and then
+overrides fields with any other yaml files provided and finally overrides with environment variables.
```

Comment:
Is this ok for production? ENV variable overrides should be disabled in enclave IMO? Or at least disabled for anything baked into the image